synapse/lib/cell.py

```diff
@@ -1267,3 +1241,33 @@ async def sync(self):
         the writes that occurred on the leader before this call.
         '''
         return
+
+    async def ps(self, user):
+        isallowed = await self.isUserAllowed(user, ('task', 'get'))
+
+        retn = []
+        for task in self.boss.ps():
+            if (task.user.iden == user) or isallowed:
+                retn.append(task.pack())
+
+        return retn
+
+    async def kill(self, user, iden):
+        perm = ('task', 'del')
+        isallowed = await self.isUserAllowed(user, perm)
+
+        logger.info(f'User [{user}] Requesting task kill: {iden}')
```

Comment:
this is another case of user object vs. user iden being passed around. since this is an iden, while correct from logging, it is difficult to triage who is doing what without using the cortex as a lookup table.